Source/JavaScriptCore/jit/ExecutableAllocator.h

```diff
@@ -296,15 +297,15 @@ static ALWAYS_INLINE void* performJITMemcpy(void *dst, const void *src, size_t n
         }
 #endif
 
-        auto ret = memcpy(dst, src, n);
+        auto ret = memcpyAtomicIfPossible(dst, src, n);
 #if ENABLE(JIT_SCAN_ASSEMBLER_BUFFER_FOR_ZEROES)
         checkForZeroes();
 #endif
         return ret;
     }
 
     // Use regular memcpy for writes outside the JIT region.
```

Comment:
This comment is confusing since we're not using regular memcpy now.

Reply:
Right. I think this comment sounds meaningless. Just dropped.